projects/angular-calendar/src/modules/common/calendar-event-actions.component.ts

```diff
@@ -15,8 +15,14 @@ import { CalendarEvent, EventAction } from 'calendar-utils';
           href="javascript:;"
           *ngFor="let action of event.actions; trackBy: trackByActionId"
           (mwlClick)="action.onClick({ event: event })"
+          (mwlKeydown)="action.onClick({ event: event })"
           [ngClass]="action.cssClass"
           [innerHtml]="action.label"
+          tabindex="0"
+          role="button"
+          [attr.aria-label]="
+            { a11yLabel: action.a11yLabel } | calendarA11y: 'actionButtonLabel'
```

Comment:
To allow maximum flexibility for the user, let's pass in the full `action` object here, in case there was some meta data on the action that the user wants to compute the label for.